src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -909,25 +909,25 @@ describe('ReactDOMServerIntegration', () => {
       });
 
       itRenders('unknown data- attributes with casing', async render => {
-        const e = await render(<div data-fooBar="true" />);
-        expect(e.getAttribute('data-fooBar')).toBe('true');
+        const e = await render(<div data-fooBar="true" />, 1);
+        expect(e.getAttribute('data-foobar')).toBe('true');
```

Comment:
Changing these calls wasn't essential but I figured I'd emphasize they are normalized by the browser anyway.